scripts/recon/train_unrolled.py

```diff
@@ -135,13 +137,39 @@ def train_unrolled(
         save = os.getcwd()
 
     start_time = time.time()
+    # Load post process model
+    if config.reconstruction.post_process.network == "DruNet":
+        from lensless.utils.image import load_drunet
+
+        post_process = load_drunet(
+            os.path.join(get_original_cwd(), "data/drunet_color.pth"), requires_grad=True
+        ).to(device)
+    elif config.reconstruction.post_process.network == "UnetRes":
+        from lensless.drunet.network_unet import UNetRes
+
+        n_channels = 3
+        post_process = UNetRes(
+            in_nc=n_channels + 1,
+            out_nc=n_channels,
+            nc=[64, 128, 256, 512],
+            nb=config.reconstruction.post_process.depth,
+            act_mode="R",
+            downsample_mode="strideconv",
+            upsample_mode="convtranspose",
+        ).to(device)
+    else:
+        post_process = None
+    pre_process = None
```

Comment:
How about raising `NotImplementedError` if pre_process != None? (inside trainable reconstruction)

Reply:
Will be irrelevant as soon as preprocess is added.